src/entities/trades/gnosis-protocol/CoWTrade.ts

```diff
@@ -124,26 +138,21 @@ export class GnosisProtocolTrade extends Trade {
    * @param chainId The chainId, defaults to Mainnet (1)
    * @returns
    */
-  public static getApi(chainId = ChainId.MAINNET) {
-    return new GnosisProtcolApi(CHAIN_ID_TO_NETWORK[chainId as keyof typeof CHAIN_ID_TO_NETWORK], Environment.Prod)
+  public static getCoWSdk(chainId = ChainId.MAINNET, cowContext?: CowContext) {
```

Comment:
getCoWSdk returns a new CowSdk.
We need to standardise capitalisation.

Reply:
Good catch. I updated this. I prefer camel-case when possible: `getCoWSDK`. There's no standardization around this.